src/target/ext/host_io.rs

```diff
@@ -25,16 +25,16 @@ bitflags! {
         /// Exclusive access.
         const O_EXCL = 0x800;
 
-        // LLDB-specific flags, as documented at:
-        // https://github.com/llvm/llvm-project/blob/ec642ceebc1aacc8b16249df7734b8cf90ae2963/lldb/include/lldb/Host/File.h#L47-L66
+        /// LLDB-specific flags, as documented at:
+        /// https://github.com/llvm/llvm-project/blob/ec642ceebc1aacc8b16249df7734b8cf90ae2963/lldb/include/lldb/Host/File.h#L47-L66
```

Comment:
Sorry, I think you misunderstood - this doc comment needs to be lifted up to the main `HostIoOpenFlags` doc comment, after line `11`.

Reply:
makes sense :)